covbot.py

```diff
@@ -597,6 +599,25 @@ def _short_location(self, location: str, length=int(12)) -> str:
         c.format = "org.matrix.custom.html"
         await e.respond(c, markdown=True, allow_html=True)
 
+    @command.new('risks', help=HELP['risks'][1])
+    @command.argument("risks", pass_raw=True, required=True)
```

Comment:
-    @command.argument("risks", pass_raw=True, required=True)
    +    @command.argument("age", pass_raw=True, required=True)

Reply:
Uh oh make sense. I fix this right now, thanks